tests/e2e/flowtest/test_selinux_relabel_solution.py

```diff
@@ -0,0 +1,239 @@
+
+log = logging.getLogger(__name__)
+
+
+class TestSelinuxrelabel(E2ETest):
+    @pytest.fixture()
+    def download_files(self):
+        kernel_url = "https://cdn.kernel.org/pub/linux/kernel/v4.x/linux-4.19.5.tar.gz"
+        download_path = ntar_loc
+        dir_path = os.path.join(os.getcwd(), download_path)
+        print(dir_path)
+        file_path = os.path.join(dir_path, "file.gz")
+        print(file_path)
+        if not os.path.exists(dir_path):
+            os.makedirs(dir_path)
+        urllib.request.urlretrieve(kernel_url, file_path)
+        print(dir_path)
+        return dir_path
+
+    @pytest.fixture()
+    def create_pvc_and_deploymentconfig_pod(self, request, pvc_factory):
+        """"""
+
+        def finalizer():
+            delete_deploymentconfig_pods(pod_obj)
+
+        request.addfinalizer(finalizer)
+
+        # Create pvc
+        pvc_obj = pvc_factory(size=20)
+
+        # Create service_account to get privilege for deployment pods
+        sa_name = helpers.create_serviceaccount(pvc_obj.project.namespace)
+
+        helpers.add_scc_policy(
+            sa_name=sa_name.name, namespace=pvc_obj.project.namespace
+        )
+
+        pod_obj = helpers.create_pod(
+            interface_type=constants.CEPHBLOCKPOOL,
+            pvc_name=pvc_obj.name,
+            namespace=pvc_obj.project.namespace,
+            sa_name=sa_name.name,
+            dc_deployment=True,
+        )
+        helpers.wait_for_resource_state(
+            resource=pod_obj, state=constants.STATUS_RUNNING
+        )
+        return pod_obj, pvc_obj
+
+    def copy_files(self, pod_obj, pvc_obj):
+        log.info(f"pod obj name---- {pod_obj.name}")
+        pod_name = pod_obj.name
+
+        log.info("cephfs pod created")
+        ocp_obj = ocp.OCP(
+            kind=constants.POD,
+            namespace=pvc_obj.project.namespace,
+        )
+        # Number of times we copy the kernel
+        copies = 3
+        tmploc = ntar_loc.split("/")[-1]
+        ocp_obj.exec_oc_cmd(
+            f"rsync {ntar_loc} {pod_name}:{constants.FLEXY_MNT_CONTAINER_DIR}",
+            timeout=300,
+        )
+        ocp_obj.exec_oc_cmd(
+            f"exec {pod_name} -- mkdir {constants.FLEXY_MNT_CONTAINER_DIR}/x"
+        )
+        for x in range(copies):
+            ocp_obj.exec_oc_cmd(
+                f"exec {pod_name} -- /bin/tar xf"
+                f" {constants.FLEXY_MNT_CONTAINER_DIR}/{tmploc}/{TARFILE}"
+                f" -C {constants.FLEXY_MNT_CONTAINER_DIR}/x/x{x}",
+                timeout=3600,
+            )
+        log.info("cephfs test files created on pod")
+
+    def calculate_md5sum(self, pod_obj, pvc_obj):
+        pod_name = pod_obj.name
+        data_path = f'"{constants.FLEXY_MNT_CONTAINER_DIR}/x"'
+        ocp_obj = ocp.OCP(
+            kind=constants.POD,
+            namespace=pvc_obj.project.namespace,
+        )
+        random_file = ocp_obj.exec_oc_cmd(
+            f"exec -it {pod_name} -- /bin/bash"
```

Comment:
This seems a duplicate code, this is already handled in data_integrity_check below